tests/test_session.py

```diff
@@ -399,7 +399,7 @@ async def refresh_post(api_options: APIOptions, user_context: Dict[str, Any]):
             "sRefreshToken": cookies["sRefreshToken"]["value"],
```

Comment:
There might be one pending change as per this slack message: https://supertokens.slack.com/archives/C03J31QAQLC/p1672825333988349